prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -180,7 +180,7 @@ public async Task<ActionResult> GetEnrolleeByUserId(Guid userId)
                 return Forbid();
             }
 
-            return RedirectToAction(nameof(GetEnrolleeById), new { enrolleeId = enrollee.Id });
+            return Redirect(Flurl.Url.Combine(PrimeConfiguration.Current.BackendUrl, "enrollees", enrollee.Id.ToString()));
```

Comment:
I have confirmed that the prod env var is:
BACKEND_URL=https://pharmanetenrolment.gov.bc.ca/api/v1/